cfme/base/ui.py

```diff
@@ -186,10 +186,10 @@ def login(self, user=None, method=LOGIN_METHODS[1]):
 
         login_view.log_in(user, method=method)
         logged_in_view.flush_widget_cache()
-        user.name = logged_in_view.current_fullname
```

Comment:
Moving this until after the `view.is_displayed` so that raises an exception before getting `None` for this property and modifying user.